docs/0.3/restapi.md

```diff
@@ -18,13 +32,44 @@ Example:
 ```bash
 curl http://127.0.0.1:8090/appmasters
 ```
+Sample Response:
+```
+{
+  "appMasters": [
+    {
+      "status": "active",
+      "appId": 1,
+      "appName": "dag",
+      "appMasterPath": "akka.tcp://app1-executor-1@127.0.0.1:62498/user/daemon/appdaemon1/$c",
+      "workerPath": "akka.tcp://master@127.0.0.1:3000/user/Worker1",
+      "submissionTime": "1425925483482",
+      "startTime": "1425925486016",
+      "user": "foobar"
+    }
+  ]
+}
+```
 
 ## GET config/app/&lt;appId&gt;
 Query the configuration of specific application appId
 
 Example:
 ```bash
-curl http://127.0.0.1:8090/config/app/0
+curl http://127.0.0.1:8090/appmaster/1
+```
```

Comment:
Yes, it will return an empty map if you have set the ClusterConfigSource in AppDescription.